redash/models.py

```diff
@@ -1369,7 +1526,7 @@ def all(cls, org, group_ids, user_id):
                  (Dashboard.user_id == user_id) |
                  ((Widget.dashboard != None) & (Widget.visualization == None))),
                 Dashboard.org == org)
-            .distinct())
```

Comment:
I had to remove this since Postgres would otherwise throw a `SELECT DISTINCT ON expressions must match initial ORDER BY expressions` error. @arikfr Do you know a better fix?

Reply:
In some places I think we just did a group by, but using distinct proved to be a better solution. Why did it start happening?

Reply:
In the method it loads the dashboard owners via a joinedload and by that tries to load the user detail column as well. When applying `distinct()` to the query it seems as if it also tries to make the join query distinct, which again fails with that error above.

Reply:
Maybe we can use a different strategy to load the users?